networking_oneview/ml2/drivers/oneview/common.py

```diff
@@ -152,19 +154,62 @@ def get_uplinkset_by_name_in_lig(oneview_client, lig_id, uplinkset_name):
     return uplinkset
 
 
-def get_logical_interconnect_group_by_id(oneview_client, lig_id):
+def get_logical_interconnect_group_by_id(lig_id):
     """Get a Logical Interconnect Group Object to a given LIG id.
 
-    :param oneview_client: a instance of the OneView Client;
     :param lig_id: the id of the Logical Interconnect Group;
     :returns: the Logical Interconnect Group object
-    :raise HPOneViewException: If was not possible to retrieve LIG;
+    :raise OneViewResourceNotFoundException: If it was not possible
+        to retrieve LIG;
     """
+    oneview_client = get_oneview_client()
     try:
         return oneview_client.logical_interconnect_groups.get(lig_id)
-    except exceptions.HPOneViewException as err:
-        LOG.error(err)
-        raise err
+    except oneview_exceptions.HPOneViewException:
+        err_msg = (
+            "Could not find a '%s' with the id '%s'"
```

Comment:
don't need to format 'Logical Interconnect Group'

Reply:
Just raise the exception?

Reply:
My bad, I understood what you meant. Put directly in the string.